src/appkit.cs

```diff
@@ -22469,7 +23170,15 @@ partial interface NSControl {
 		bool AllowsExpansionToolTips { get; set; }
 	}
 
-	partial interface NSMatrix {
+	[Protocol]
+	interface NSViewToolTipOwner
```

Comment:
availability 10.14 according to docs -> https://developer.apple.com/documentation/appkit/NSViewToolTipOwner?language=objc